Automattic/wp-calypso
# Make the git hooks go faster by not needing to install node_modules if they are outdated.

This makes the Git hooks faster. Now, it only requires that there is _something_ in `node_modules` for the hooks to run, the dependencies don't need to be updated.

To test:
* `touch npm-shrinkwrap.json`
* Try to commit something with a `lint` error.

Result: The hook should fail fast.

To test (optional):
* Nuke your `node_modules` folder
* Try to commit something correct

Result: The hook should (eventually) succeed.

Note: To test the hooks, you can just run `npm run precommit` or `npm run prepush`, if you don't want to mess with Git branches, resetting them, etc.
